tasks/copy_to_glacier/README.md

```diff
@@ -238,7 +239,8 @@ often be derived from the collection configuration in Cumulus as seen below:
           "event.$": "$",
           "task_config": {
             "multipart_chunksize_mb": "{$.meta.collection.multipart_chunksize_mb"},
-            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}"
+            "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
+            "orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"
```

Comment:
Need to change `"orcaDefaultBucketOverride": "{$.meta.collection.meta.orca_default_bucket}"` to 
`"orcaDefaultBucketOverride": "{$.meta.collection.meta.orcaDefaultBucketOverride}"` to be consistent with our other items in collection configuration.

Reply:
Fixed